src/e2e/puppeteer/__tests__/render-thoughts.ts

```diff
@@ -14,6 +16,20 @@ expect.extend({
 
 vi.setConfig({ testTimeout: 60000, hookTimeout: 20000 })
 
+/** Returns a snapshot for render-thoughts/superscript. */
+const superscriptSnapshot = async () => {
+  await paste(`
+    - a
+      - m
+    - b
+      - m
+  `)
+
+  await press('ArrowUp')
+
+  return await screenshot()
```

Comment:
FYI `return await` can always be simplified to `return`. The `await` is implied by the async function.

Reply:
Good to know, thanks!